deploy/runtime/startup.lk

```diff
@@ -35,6 +35,7 @@ addconfig( 'Generic System', ['Residential', 'Commercial', 'Third Party', 'Host
 addconfig( 'PV Battery', ['Residential', 'Commercial', 'Third Party', 'Host Developer', 'Single Owner', 'Merchant Plant', 'Leveraged Partnership Flip', 'All Equity Partnership Flip', 'Sale Leaseback'] );
 addconfig( 'PVWatts Battery', ['Residential', 'Commercial', 'Third Party', 'Host Developer'] );
 addconfig( 'Generic Battery', ['Residential', 'Commercial', 'Third Party', 'Host Developer', 'Single Owner', 'Merchant Plant', 'Leveraged Partnership Flip', 'All Equity Partnership Flip', 'Sale Leaseback'] );
+addconfig( 'Retired', ['None']);
```

Comment:
I believe these changes need to be merged to align with the changes in startup.

Reply:
@mjprilliman, sorry, yes please keep these and check the merge closely. Github is notorious for screwing up text file merges!